### PR TITLE
tools/zep_dispatch: allow to pin nodes to MAC address

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

Positions in the topology are handed out first come first serve.
This makes it hard to have a reproducible setup if the ZEP dispatcher is restarted (e.g. to modify the topology) or if a node is restarted.

Fix this by allowing to specify a MAC address for each node.
If a node with a pinned MAC address connects, it will always be assigned to the same topology node.

### Testing procedure

Create a topology file with nodes pinned to specific MAC addresses:

```
A := AA:AA:AA:AA:AA:AA:AA:AA
B := BB:BB:BB:BB:BB:BB:BB:BB
C := CC:CC:CC:CC:CC:CC:CC:CC

A B
B C
```

Run the ZEP dispatcher with this file:

    make -C dist/tools/zep_dispatch TOPOLOGY=test.topo run

Now start three instances of `native` with the set IDs:

    make -C examples/gnrc_networking USE_ZEP=1 ZEP_MAC=AA:AA:AA:AA:AA:AA:AA:AA all term
    make -C examples/gnrc_networking USE_ZEP=1 ZEP_MAC=BB:BB:BB:BB:BB:BB:BB:BB all term
    make -C examples/gnrc_networking USE_ZEP=1 ZEP_MAC=CC:CC:CC:CC:CC:CC:CC:CC all term

Node B should be able to ping it's two neighbors:

```
> ping ff02::1
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=0 ttl=64 rssi=24672 dBm time=0.564 ms
12 bytes from fe80::a8aa:aaaa:aaaa:aaaa%7: icmp_seq=0 ttl=64 rssi=24672 dBm time=0.620 ms (DUP!)
12 bytes from fe80::a8aa:aaaa:aaaa:aaaa%7: icmp_seq=1 ttl=64 rssi=24672 dBm time=0.491 ms
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=1 ttl=64 rssi=24672 dBm time=0.528 ms (DUP!)
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=2 ttl=64 rssi=24672 dBm time=0.509 ms

--- ff02::1 PING statistics ---
3 packets transmitted, 3 packets received, 2 duplicates, 0% packet loss
round-trip min/avg/max = 0.491/0.542/0.620 ms
```

Now restart a single node. It should again take it's assigned place in the topology. 

But we can also modify the topology and restart the dispatcher:

```
A := AA:AA:AA:AA:AA:AA:AA:AA
B := BB:BB:BB:BB:BB:BB:BB:BB
C := CC:CC:CC:CC:CC:CC:CC:CC

B C
C A
```

Now B should only reach node C:

```
> ping ff02::1
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=0 ttl=64 rssi=24672 dBm time=0.399 ms
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=1 ttl=64 rssi=24672 dBm time=0.543 ms
12 bytes from fe80::cecc:cccc:cccc:cccc%7: icmp_seq=2 ttl=64 rssi=24672 dBm time=0.474 ms

--- ff02::1 PING statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 0.399/0.472/0.543 ms
```

### CODE REVIEW

File: dist/tools/zep_dispatch/l2util.c [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../sys/net/link_layer/l2util/l2util.c
```
Comment (bergzand) [inline dist/tools/zep_dispatch/l2util.c:1]: I would prefer to have these in the Makefile, similar to how `ieee802154.c` is included in the build now. Is there anything against that?

Comment (benpicco) [reply dist/tools/zep_dispatch/l2util.c]: fine with me

**Comment by benpicco** [comment]
ping @bergzand

**Review by maribu — APPROVED** [review]
ACK. Code looks good to me and I'm trusting @benpicco's testing.

**Comment by benpicco** [comment]
Thank you for the review!

**Pull request merged**